### PR TITLE
Translate support-for-ruby-1-9-3-has-ended (ko)

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

**Comment by marocchino** [comment]
@namsk review please.

File: ko/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+author: "Olivier Lacan"
+translator: "marocchino"
+date: 2015-02-23 00:00:00 +0000
+lang: ko
+---
+
+오늘(2015년 2월 23일)부터 루비 1.9.3의 모든 지원이 종료됩니다. 루비 최신 버전에
+적용하는 버그 수정, 보안 패치는 더는 1.9.3에 백 포트되지 않습니다.
+
+이 지원 종료는 [1년 전부터 공지](https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/)했었습니다.
```
Comment (JuanitoFatas) [inline ko/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: Korean translation is available: https://www.ruby-lang.org/ko/news/2014/01/10/ruby-1-9-3-will-end-on-2015/
:wink:

Comment (marocchino) [inline ko/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: Thank you! :yum:

**Comment by hsbt** [comment]
@marocchino @namsk Thanks always!

**Comment by namsk** [comment]
@hsbt Well, You accept me as a member of i18n team and @marocchino makes Korean page breathe again.
I am not a professional translator, and I am not a good ruby programmer.
I just don't want Korean page to stay "unmaintained". 

Thanks for accepting me, and thanks for translating @hsbt and @marocchino :)

**Pull request merged**